docs/examples/Power Flow Example.ipynb

```diff
@@ -1372,7 +1384,7 @@
       "\n",
       "Failed scenarios: [3 7]\n",
       "Succeeded scenarios: [0 1 2 4 5 6 8 9]\n",
-      "Error messages: ['The id cannot be found: 1000\\n', 'Iteration failed to converge after 20 iterations! Max deviation: 5.81174e+85, error tolerance: 1e-08.\\n']\n"
+      "Error messages: ['The id cannot be found: 1000\\n', 'Sparse matrix error, possibly singular matrix!\\nIf you get this error from state estimation, it might mean the system is not fully observable, i.e. not enough measurements.\\nIt might also mean that you are running into a corner case where PGM cannot resolve yet.See https://github.com/PowerGridModel/power-grid-model/issues/864.']\n"
```

Comment:
😮 interesting... i did not think we changed behavior... can you please investigate in a separate branch which version caused this difference?... or maybe the change happened when we improved the sparse matrix calculator?

it's at least good to have an idea of what caused this

Reply:
Example was last modified pre LU solver bug. I suppose this power flow gets classified first as SparseMatrix before actually diverging.